dicoogle/src/main/java/pt/ua/dicoogle/server/ControlServices.java

```diff
@@ -50,7 +50,7 @@
     
     private ControlServices()
     {
-        TaskManager taskManager = new TaskManager(4);
+        TaskManager taskManager = new TaskManager(Integer.parseInt(System.getProperty("dicoogle.taskManager.nThreads", "4")));
```

Comment:
This looks superfluous. Don't we want a single task manager? This one is just dropped out of scope when leaving the constructor.